taf/repository_tool.py

```diff
@@ -698,11 +780,12 @@ def map_signing_roles(self, target_filenames):
         is expected to be relative to the targets directory. It can be defined as a glob
         pattern.
         """
+        if delegations_info_fn is None:
```

Comment:
Could you add docstring for `delegations_info_fn`? Or no needed if we could implement a method to return a repository at some other state (commit sha).

Reply:
I think that it makes sense to have such a functionality, unless it adds to much complexity to our code base since we must avoid modifying TUF. I'll try to implement something, but will timebox it to say 2-3 hours since we don't really need this functionality in other places at the moment. If I don't make much progress, I'll create an issue and we can address it when there is a greater need.